Src/FluentAssertions/Collections/SelfReferencingCollectionAssertions.cs

```diff
@@ -439,15 +439,17 @@ public AndConstraint<TAssertions> EndWith(IEnumerable<T> expectation, string bec
         /// <summary>
         /// Asserts that the collection contains some extra items in addition to the original items.
         /// </summary>
-        /// <param name="expectedItemsList">An <see cref="IEnumerable{T}"/> of expectation items.</param>
-        /// <param name="additionalExpectedItems">Additional items that are expectation to be contained by the collection.</param>
-        public AndConstraint<TAssertions> Contain(IEnumerable<T> expectedItemsList,
-            params T[] additionalExpectedItems)
+        /// <param name="expected">An <see cref="IEnumerable{T}"/> of expectation items.</param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because"/>.
+        /// </param>
+        public AndConstraint<TAssertions> Contain(IEnumerable<T> expected, string because = "", params object[] becauseArgs)
```

Comment:
I don't get this change. You seem to be replacing an overload that takes an additional set of items to include in the expected collection by the overload that takes a reason.